.github/workflows/push.yml

```
@@ -173,6 +177,8 @@ jobs:
       BUILD_CHUNKS: ${{ steps.build_map.outputs.BUILD_CHUNKS }}
       IMAGES: ${{ steps.deploy_map.outputs.IMAGES }}
       node-modules-hash: ${{ steps.calculate_node_modules_hash.outputs.node-modules-hash }}
```

actionlint (1.7.4)

property "calculate_node_modules_hash" is not defined in object type {build_map: {conclusion: string; outcome: string; outputs: {string => string}}; deploy_map: {conclusion: string; outcome: string; outputs: {string => string}}; docker-deps: {conclusion: string; outcome: string; outputs: {string => string}}; docker_tags: {conclusion: string; outcome: string; outputs: {string => string}}; git_nx_base: {conclusion: string; outcome: string; outputs: {string => string}}; git_nx_head: {conclusion: string; outcome: string; outputs: {string => string}}; git_refs: {conclusion: string; outcome: string; outputs: {string => string}}; nodejs_image: {conclusion: string; outcome: string; outputs: {string => string}}; restore-cache: {conclusion: string; outcome: string; outputs: {string => string}}; slack: {conclusion: string; outcome: string; outputs: {}}; test_projects: {conclusion: string; outcome: string; outputs: {string => string}}; unaffected: {conclusion: string; outcome: string; outputs: {string => string}}; yarn-cache-dir-path: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)